PostHog/PostHogConfig.swift

```diff
@@ -202,7 +199,7 @@ public typealias BeforeSendBlock = (PostHogEvent) -> PostHogEvent?
         return integrations
     }
 
-    var _surveys: Bool = false // swiftlint:disable:this identifier_name
+    var _surveys: Bool = true // swiftlint:disable:this identifier_name
```

Comment:
i would still keep it as `false` by default, so people have to turn it on

Reply:
or should it be enabled by default? good question

Reply:
have to fix here as well https://posthog.com/docs/libraries/ios#all-configuration-options

> /**
>  * Enable Surveys. (off/false by default)
>  * Experimental feature under @_spi(Experimental)
>  */
> configuration.surveys = false